rules/linux/lnx_system_shutdown_reboot.yml

```diff
@@ -0,0 +1,40 @@
+title: 'System Shutdown/Reboot'
+id: 4cb57c2f-1f29-41f8-893d-8bed8e1c1d2f
+status: experimental
+description: 'Adversaries may shutdown/reboot systems to interrupt access to, or aid in the destruction of, those systems.'
+  # For this rule to work execve auditing must be configured
+  # Example config (place it at the bottom of audit.rules)
+  # -a always,exit -F arch=b32 -S execve -k execve
+  # -a always,exit -F arch=b64 -S execve -k execve
+author: 'Igor Fits, oscd.community'
+date: 2020/10/15
+references:
+    - hhttps://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1529/T1529.md
+logsource:
+    product: linux
+    service: auditd
+detection:
+    selection1:
+       type: 'EXECVE'
+       keywords|contains:
+              - 'shutdown'
+              - 'reboot'
+              - 'halt'
+              - 'poweroff'
+    selection2:
+       type: 'EXECVE'
+       keywords|contains:
+              - 'init'
+              - 'telinit'
+    selection3:
+        type: 'EXECVE'
+        keywords|contains:
+              - '0'
+              - '6'
+    condition: selection1 or (selection2 and selection3)
+falsepositives:
+    - 'Legitimate administrative activity'
+level: high
```

Comment:
I think this is too high, as these events are very usual in Linux environments.

Reply:
discussion here https://github.com/SigmaHQ/sigma/pull/1362#discussion_r598336458